lib/shared/src/chat/preamble.ts

```diff
@@ -9,7 +9,8 @@ const DEFAULT_PREAMBLE = ps`You are Cody, an AI coding assistant from Sourcegrap
  * produce code blocks that we can associate with existing file paths.
  * We want to read these file paths to support applying code directly to files from chat.
  */
-const SMART_APPLY_PREAMBLE = ps`If your answer contains fenced code blocks in Markdown, include the relevant full file path in the code block tag using this structure: \`\`\`$LANGUAGE:$FILEPATH\`\`\`.`
+const SMART_APPLY_PREAMBLE = ps`If your answer contains fenced code blocks in Markdown, include the relevant full file path in the code block tag using this structure: \`\`\`$LANGUAGE:$FILEPATH\`\`\`
+For executable terminal commands: enclose in a Markdown code block with the language tag "bash".`
```

Comment:
If Cody provides multiple commands then there are sometimes docstrings included:
![Screenshot 2024-09-27 131843](https://github.com/user-attachments/assets/bc124a32-85b3-4ef0-808d-f0822b15cb26)

This will not affect the execution of the commands, but could lead to potential issues:
```suggestion
For executable terminal commands: enclose in a Markdown code block with the language tag "bash". Avoid including documentation to the executable commands.`
```

Reply:
Good point! I've updated the prompt to include this rule with minor adjustment that hopefully would work with most LLMs 🙏